src/main/java/com/arcao/fontcreator/FontCreator.java

```diff
@@ -63,7 +63,7 @@ public static void main(String args[]) {
             FontConverterV3 fontConverter = new FontConverterV3(font, charset);
 
             StringBuilder builder = new StringBuilder();
-            fontConverter.printFontData(builder);
+            fontConverter.printFontData(builder,arguments.yoffset());
```

Comment:
Please reformat the line. Space should be after comma.

Reply:
Done!